lib/src/converter/protobuf_converter.dart

```diff
@@ -60,13 +60,16 @@ class ProtobufConverterImpl implements ProtobufConverter {
       message.serviceData.map((entry) => Uuid(entry.serviceUuid.data)),
       message.serviceData.map((entry) => Uint8List.fromList(entry.data)),
     );
+    final serviceUuids =
```

Comment:
I would like to see a unittest for this functionality

Reply:
Done